test/metabase/query_processor_test/timezones_test.clj

```diff
@@ -12,6 +13,32 @@
             [metabase.util.honeysql-extensions :as hx]
             [toucan.db :as db]))
 
+(defn- do-with-temporary-report-timezone
+  "Implementation for the `with-temporary-humanization-strategy` macro."
+  [value thunk]
+  (let [original-value (driver/report-timezone)]
+    (try
+      (driver/report-timezone value)
+      (testing (colorize/blue (format "\nSetting :report-timezone = %s\n" (pr-str value)))
+        (thunk))
+      (catch Throwable e
+        (throw (ex-info (str "Error in with-temporary-report-timezone: " (ex-message e))
+                        {:setting :report-timezone
+                         :value   value}
+                        e)))
+      (finally
+        (driver/report-timezone original-value)))))
+
+(defmacro with-temporary-report-timezone
+  "Temporarily set the `report-timezone` setting to the provided value, execute body, and re-establish the original
+  setting. This should be used instead of `metabase.test.util/with-temporary-setting-values` to ensure that cached
+  database resources are released when restoring the setting (see `driver/notify-all-databases-updated`).
+
+     (with-temporary-report-timezone :simple
```

Comment:
I don't think this example is relevant? I'm guessing this was accidentally copied from the humanization strategy macro docstring